plane/source/docs/automatic-tuning-with-autotune.rst

```diff
@@ -135,7 +135,9 @@ coping well with wind, or not holding altitude well. Keep flying in
 AUTOTUNE mode well past the point where you think the plane is flying
 well.
 
-If you have an OSD or Ground Control Station, then messages for D and P tuning phases for the axis, as well as a "Tuning Completed" message will be displayed. Once tuning is completed, it is saved.
+If you have an OSD or Ground Control Station, then messages for D and P tuning phases for the axis, as well as a "Tuning Completed" message will be displayed.
```

Comment:
```suggestion
If you have an OSD or Ground Control Station, then messages for D and P tuning phases for the axis, as well as a "Tuning Completed" message will be displayed. The values are saved as the tuning progresses and P and D values are saved automatically. The FF (feed-forward) values are continuously adjusted as long as the plane remains in AUTOTUNE mode.
```

Reply:
Are you sure? That means there is no way to undo an AUTOTUNE in plane?

Reply:
I is applied based on FF